cg/meta/report/api.py

```diff
@@ -126,14 +127,21 @@ def get_report_data(self, case_id: str, analysis_date: datetime) -> ReportModel:
         case = self.status_db.family(case_id)
         analysis = self.status_db.analysis(case, analysis_date)
         analysis_metadata = self.analysis_api.get_latest_metadata(case.internal_id)
+
+        if not analysis_metadata:
```

Comment:
I find it a bit inconsistent that we check if got any `analysis_metadata` from the method `get_latest_metadata` but the other get-methods don't get this type of treatment. Would it make sense to change this so it is all or nothing?

Reply:
👍